runtime/tests/interpreter/builtinfunctions_test.go

```diff
@@ -474,6 +474,13 @@ func TestInterpretToBigEndianBytes(t *testing.T) {
 			"9223372036854775808":  {128, 0, 0, 0, 0, 0, 0, 0},
 			"18446744073709551615": {255, 255, 255, 255, 255, 255, 255, 255},
 		},
+		"Word128": {
+			"0":   {0},
+			"42":  {42},
+			"127": {127},
+			"128": {128},
+			"200": {200},
+		},
```

Comment:
Maybe also add some tests for larger values here, and while at it improve `UInt128` too

Reply:
Done